lib/web_ui/test/canvaskit/picture_test.dart

```diff
@@ -68,6 +68,61 @@ void testMain() {
       expect(data!.lengthInBytes, 10 * 15 * 4);
       expect(data.buffer.asUint32List().first, color.value);
     });
-  // TODO(hterkelsen): https://github.com/flutter/flutter/issues/60040
+
+    test('cullRect bounds are tight', () async {
```

Comment:
Should we also test cases where the estimated size blows up to infinity? E.g. when the app does `canvas.drawColor` or `canvas.drawPaint`? What's the expected value of `cullRect`, and how should it be interpreted by the engine code?

Reply:
In this case it should default to the cullRect passed in when the PictureRecorder begins recording (we use ui.Rect.largest). I added a test to confirm this.